src/httpServer/server.ts

```diff
@@ -223,11 +223,9 @@ export class DBOSHttpServer {
   static registerCancelWorkflowEndpoint(dbosExec: DBOSExecutor, router: Router) {
     const workflowCancelUrl = '/workflows/:workflow_id/cancel';
     const workflowCancelHandler = async (koaCtxt: Koa.Context) => {
-      // eslint-disable-next-line  @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access
-      const workflowId = koaCtxt.params.workflow_id;
+      const workflowId = (koaCtxt.params as { workflow_id: string }).workflow_id;
```

Comment:
Not sure this cast is always safe. Why don't we do something like:

```
  if (typeof workflow_id !== "string") {
    koaCtxt.throw(400, "Invalid workflow_id");
    return;
  }

```

Reply:
That check doesn't actually do anything :)

Reply:
🤔